From 9fd860fed9ad1e468c09e1925f8ff40defb131bc Mon Sep 17 00:00:00 2001
From: Fabian Leutenegger <fabian.leutenegger@bluewin.ch>
Date: Wed, 23 Feb 2022 00:40:42 +0100
Subject: [PATCH 13/15] Screenshot: Remove text from expanded screenshot chip

- We are running out of space in portrait after
  23205: Screenshot: Add delete action chip intent | https://gerrit.aospa.co/c/AOSPA/android_frameworks_base/+/23205

Change-Id: I8967e1c5925701a84bca81d7f15b3d0457c2bb0f
---
 .../src/com/android/systemui/screenshot/ScreenshotView.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotView.java b/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotView.java
index a0e761096e75..167e1deda813 100644
--- a/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotView.java
+++ b/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotView.java
@@ -723,7 +723,7 @@ public class ScreenshotView extends FrameLayout implements
             mPendingInteraction = PendingInteraction.PREVIEW;
         });
 
-        mScrollChip.setText(mContext.getString(R.string.screenshot_scroll_label));
+        //mScrollChip.setText(mContext.getString(R.string.screenshot_scroll_label));
         mScrollChip.setIcon(Icon.createWithResource(mContext,
                 R.drawable.ic_screenshot_scroll), true);
         chips.add(mScrollChip);
-- 
2.34.1

